README.md

```diff
@@ -22,18 +22,21 @@ the boot2docker ISO.
 <a name="building-the-centos-iso"></a>
 ## Building the CentOS ISO
 
-The following contains instructions on how to build the default (CentOS based) ISO.
+The following contains instructions on how to build the default (CentOS7 based) ISO.
 If you are able to install [livecd-tools](https://github.com/rhinstaller/livecd-tools)
-directly on your machine, you can use the [CentOS/Fedora](#on-centosfedora) instructions.
+directly on your machine, you can use the [CentOS](#on-centos) instructions.
 
-If you don't have _livecd-tools_, follow the
-[hosts without livecd-tools](#on-hosts-without-livecd-tools-os-x-windows-) instructions.
+If you don't have _livecd-tools or using different linux distro other than centos_, follow the
+[hosts other than CentOS-7](#non-centos7-hosts) instructions.
 
-<a name="on-centosfedora"></a>
-### On CentOS/Fedora
+<a name="on-centos"></a>
+### On CentOS
 
 <a name="prerequisites"></a>
 #### Prerequisites
+* Update your system before start and if there is kernel update then reboot your system to activate latest kernel.
+
+        $ yum update -y
 
 * [livecd-tools](https://github.com/rhinstaller/livecd-tools)
```

Comment:
We need to add docker to the prerequisite. Also a pointer to the cause i.e. https://bugzilla.redhat.com/show_bug.cgi?id=1303565 would be great.